### DISCUSSION TITLE
Welcome to hftbacktest Discussions!

### DISCUSSION THREAD

**Discussion by nkaz001** [nkaz001/hftbacktest#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mrfranken** [comment]
hi, I concentrate on your repo for long time, will you plan to create more example or tutorial for how to using. BR

**Reply by nkaz001** [reply]
Definitely, I have a plan to add more examples and tutorials to provide a better understanding of this library. Do you have any thoughts on where supplementary material might be needed?
